articles/openshift/oshift-ref-no_proxy.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Specifying endpoints to avoid being proxied in deployments with government community network connectivity | UKCloud Ltd
+description: Provides information regarding proxy environment variables in v3.11 OpenShift clusters deployed with government community network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Specifying endpoints to avoid being proxied in deployments with government community network connectivity
+toc_fullpath: Reference/oshift-ref-no_proxy.md
```

Comment:
Ideally, the filename should use a hyphen instead of an underscore: oshift-ref-no-proxy.md
(I know that the variable uses an underscore, but for the purposes of following file naming conventions, we should use a hypen here)